lib/jellyfishService.js

```diff
@@ -153,60 +164,71 @@ function createServer(serverConfig, mongoClient, seagullClient, userApiClient, g
   app.use(bodyparser.json({ limit: '4mb' }));
   app.use(errorHandler({ dumpExceptions: true, showStack: true }));
 
-  app.get('/info', function(request, response) {
+  app.get('/info/?:userId?', function (request, response) {
     log.info('Handling versions request');
 
+    var uploaderDestination = 'jellyfish';
+    if (request.params.userId) {
+      log.info('using user id to detrimine upload destination');
+      if (serverConfig.uploader.platformUsers.includes(request.params.userId)) {
```

Comment:
We should map user id to lowercase and trim whitespace for comparison as we do in `env.js`, just in case.

We should map the user id to lowercase and trim whitespace (just in case).